src/preload-hover.js

```diff
@@ -19,18 +19,28 @@ export default class PreloadHover {
     if (!domScopes) { throw new Error('domScopes must be provided.'); }
     if (domLinks != 'local' && domLinks != 'external' && domLinks != 'both') { throw new Error('linkType must be local, external or both.'); }
     const head = document.getElementsByTagName('head')[0];
+    const links = document.getElementsByTagName('a');
+    let uniqueLinks = uniqBy(links, 'href');
 
     //create function
-    function loadAttr(preload, linkHref) {
+    const loadAttr = (preload, linkHref) => {
       preload.setAttribute('rel', 'preload');
       preload.setAttribute('href', linkHref.href);
       head.appendChild(preload);
     }
 
+    document.body.addEventListener('mouseover', (e) => {
```

Comment:
We should apply this to the provided `domScopes`, rather than hardcoding to `document.body`. For now, don't worry about potential overlaps of multiple `domScopes` values, we can resolves that later if needed.

It looks like we already have that below.